client/src/pages/Contest/Contest.tsx

```diff
@@ -48,21 +49,33 @@ export default function Contest(): JSX.Element {
       imageFiles: string[];
     }>,
   ) => {
-    createContest(title, description, prizeAmount, createDeadlineDate(date, time, timeZone), imageFiles).then(
-      (data) => {
-        if (data.error) {
-          setSubmitting(false);
-          updateSnackBarMessage(data.error.message);
-        } else if (data.success) {
-          history.push('/dashboard');
-          updateSnackBarMessage('Created Contest Successfully');
-        } else {
-          console.error({ data });
-          setSubmitting(false);
-          updateSnackBarMessage('An unexpected error occured. Please try again');
-        }
-      },
+    const contest = await createContest(
+      title,
+      description,
+      prizeAmount,
+      createDeadlineDate(date, time, timeZone),
+      imageFiles,
     );
+
+    if (contest.error) {
+      setSubmitting(false);
+      updateSnackBarMessage(contest.error.message);
+    } else if (contest.success) {
+      const stripeUser = await stripeCustomer();
+
+      if (stripeUser.existingStripeCustomer) {
+        setSubmitting(false);
+        updateSnackBarMessage('Card on file will be charged when you select the winner!');
+        history.push('/dashboard');
```

Comment:
I guess you would want to use the '/discover' route here as opposed to the dashboard route?

Reply:
So my understanding that if a user creates a contest, the user should get directed to the dashboard where they can see their recently created contest.